modules/apps/portal-search/portal-search-web/src/main/resources/content/Language.properties

```diff
@@ -158,6 +158,8 @@ search-results-template-key-default=Search Results Template Key Default
 search-results-web-template-configuration-name=Search Results Web Template Configuration Name
 search-web-configuration-name=Search Web
 set-up-its-destination-to-make-it-visible=Set up its destination to make it visible.
+show-results-from-staged-sites=Show results from staged sites when scope is Everything
+show-results-from-staged-sites-help=When local staging is activated for the current site and the search is performed on the staged page, the staged contents will appear in the results if the scope is set to 'Everything' and this option is enabled.
```

Comment:
I guess it's correct to just say "staging" and not "local staging", right?

Reply:
I think that should be specifically "local staging", because if we have remote staging activated, even though the search scope is set to Everything, I reckon it won't search in the remote site.